pkg/api/vlabs/validate.go

```diff
@@ -768,17 +768,6 @@ func (a *AgentPoolProfile) validateCustomNodeLabels(orchestratorType string) err
 	return nil
 }
 
-func (a *AgentPoolProfile) validateKubernetesDistro() error {
-	switch a.Distro {
-	case AKS, AKS1804, Ubuntu1804:
-		if a.IsNSeriesSKU() {
-			return errors.Errorf("The %s VM SKU must use the %s or %s Distro as they require the docker-engine container runtime with Ubuntu 16.04-LTS", a.VMSize, AKSDockerEngine, Ubuntu)
```

Comment:
since we're removing the "GPU needs to use ubuntu 16.04" validation let's run a test to verify that GPU + 18.04 works

Reply:
verified